scripts/prebuild.ts

```diff
@@ -0,0 +1,63 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import axios from 'axios';
+
+console.warn('Prebuilding...');
+
+const rootDir = path.dirname(__dirname);
+
+function idPoorPrebuild(country: string): void {
+  const dataDir = path.join(rootDir, `public/data/${country}`);
+  const file = path.join(dataDir, 'idpoor.json');
+
+  switch (country) {
+    case 'cambodia':
+      console.warn('ID Poor Cambodia');
+
+      fs.access(file, fs.constants.F_OK, (fileMissingError: any) => {
+        if (!fileMissingError) {
+          // fetch data from idpoor api
+          axios
+            .get('http://localhost/idpoor/data')
```

Comment:
Out of curiosity, does this just work? Does anything need to start up before this works? If so, we should document it somewhere